.gitignore

```diff
@@ -12,3 +12,4 @@
 *.egg-info
 *.po~
 Dockerfile_deployment_environment
+package-lock.json
```

Comment:
Where is this file provided?

Reply:
`pakage-lock.json` file is generated by recent (5.4.0?) versions of npm when doing an install. It is meant to build a tree of dependencies, or something such.